src/Authorization.php

```diff
@@ -23,10 +23,15 @@ final class Authorization
     private const MERCURE_AUTHORIZATION_COOKIE_NAME = 'mercureAuthorization';
 
     private $registry;
+    private $jwtLifetime;
 
-    public function __construct(HubRegistry $registry)
+    /**
+     * @param int|null $jwtLifetime If not null, an "exp" claim is always set to now + $jwtLifetime (in seconds)
+     */
+    public function __construct(HubRegistry $registry, int $jwtLifetime = null)
```

Comment:
```suggestion
    public function __construct(HubRegistry $registry, ?int $jwtLifetime = null)
```
For consistency?

Reply:
We can even set a default value then, it will be safer.